resources/views/bread/browse.blade.php

```diff
@@ -241,50 +241,58 @@
-    @endif
-    <script>
-        $(document).ready(function () {
-            @if (!$dataType->server_side)
-                var table = $('#dataTable').DataTable({!! json_encode(
+<!-- DataTables -->
+@if(!$dataType->server_side && config('dashboard.data_tables.responsive'))
+    <script src="{{ voyager_asset('lib/js/dataTables.responsive.min.js') }}"></script>
+@endif
+<script>
+    $(document).ready(function () {
+                @if (!$dataType->server_side)
+        var table = $('#dataTable').DataTable({!! json_encode(
                     array_merge([
                         "order" => [],
                         "language" => __('voyager::datatable'),
                         "columnDefs" => [['targets' => -1, 'searchable' =>  false, 'orderable' => false]],
                     ],
                     config('voyager.dashboard.data_tables', []))
                 , true) !!});
-            @else
-                $('#search-input select').select2({
-                    minimumResultsForSearch: Infinity
-                });
-            @endif
+        @else
+        $('#search-input select').select2({
+            minimumResultsForSearch: Infinity
+        });
+        @endif
 
-            @if ($isModelTranslatable)
-                $('.side-body').multilingual();
-                //Reinitialise the multilingual features when they change tab
-                $('#dataTable').on('draw.dt', function(){
-                    $('.side-body').data('multilingual').init();
-                })
-            @endif
-            $('.select_all').on('click', function(e) {
-                $('input[name="row_id"]').prop('checked', $(this).prop('checked'));
-            });
+        @if ($isModelTranslatable)
+        $('.side-body').multilingual();
+        //Reinitialise the multilingual features when they change tab
+        $('#dataTable').on('draw.dt', function(){
+            $('.side-body').data('multilingual').init();
+        })
+        @endif
+        $('.select_all').on('click', function(e) {
+            $('input[name="row_id"]').prop('checked', $(this).prop('checked'));
         });
+    });
 
 
-        var deleteFormAction;
-        $('td').on('click', '.delete', function (e) {
-            $('#delete_form')[0].action = '{{ route('voyager.'.$dataType->slug.'.destroy', ['id' => '__id']) }}'.replace('__id', $(this).data('id'));
-            $('#delete_modal').modal('show');
-        });
-    </script>
+    var deleteFormAction;
+    $('td').on('click', '.delete', function (e) {
+        $('#delete_form')[0].action = '{{ route('voyager.'.$dataType->slug.'.destroy', ['id' => '__id']) }}'.replace('__id', $(this).data('id'));
+        $('#delete_modal').modal('show');
+    });
+
+    $('tr.js-row').on('dblclick', function (e) {
+        var doubleClickLocation = "{{lcfirst($dataType->display_name_plural)}}" + "/" + $(this).data('id');
+        @can('edit', app($dataType->model_name))
+            doubleClickLocation += "/edit";
```

Comment:
I think varying the click action like this might be confusing to end-users.

I believe the `read` view also has an edit button, can we simply link them to the read page and they can edit if desired?

I'm open to suggestions on this.

Reply:
I thought about this as well. The thing that made me choose to do it this way is the speed of usage of the admin panel for the end user. If I'm double clicking a row and I do have edit rights, I'm almost never interested in just showing the record, but almost always in directly editing it. If we always redirect the user to the `read` view, the user who wants to edit the record always has to perform another click. The way it's made right now is faster.

Reply:
This kind of dilemma could be an option from the user and we could implement both...? Just a simple suggestion.
My usage would be to have a better global view of a record in table with lot of column. I don't specially want to edit it. But as the view mode isn't really different as the edit, it doesn't matter much to me...